install/kubernetes/cilium/README.md

```diff
@@ -2,355 +2,320 @@
 
 ![Version: 1.9.90](https://img.shields.io/badge/Version-1.9.90-informational?style=flat-square) ![AppVersion: 1.9.90](https://img.shields.io/badge/AppVersion-1.9.90-informational?style=flat-square)
```

Comment:
I was wondering what these versions represent? :confused: 
Shouldn't these be `1.9.0`?

Reply:
In the tree during v1.10 development, we use the version number v1.9.90 today to denote "a version that may be much greater than v1.9.x but still a bit less than v1.10.0".